### PR TITLE
Allow control over app version to download directly

### PR DESCRIPTION
**Pull request by esoergel**
http://manage.dimagi.com/default.asp?226538#1151331
@emord

### CODE REVIEW

**Comment by esoergel** [comment]
@amstone326 FYI

File: corehq/apps/app_manager/views/cli.py [hunk 1]
```diff
@@ -31,18 +41,61 @@ def app_to_json(app):
...
-    msg = "You must specify `app_id` in your GET parameters"
-    return json_response({'status': 'error', 'message': msg}, status_code=400)
+    """
+    You must specify an app_id, and you may specify either 'version' or 'latest'
+    latest can be one of:
+        release: Latest starred version
+        build: Latest version regardless of star
+        save: Latest saved version of the application (even without a build)
+    If 'version' and 'latest' aren't specified it will default to latest save
+    """
```
Comment (esoergel) [inline corehq/apps/app_manager/views/cli.py]: @amstone326 This is the closest thing to documentation I've written thus far (the error messages should be pretty explanatory though).  Do you know of a related place on confluence to stick this?

Comment (amstone326) [inline corehq/apps/app_manager/views/cli.py]: @esoergel Hey, can you specify what the format of the URL is meant to be now? I can find somewhere on confluence to add this once I have that.

Comment (esoergel) [inline corehq/apps/app_manager/views/cli.py]: These arguments are all done as url parameters, so with the same base URL of `*.commcarehq.org/a/mydomainname/apps/api/download_ccz/`

Get version 5: `?app_id=<app_id>&version=5`
Get latest build: `?app_id=<app_id>&latest=build`
Get latest release: `?app_id=<app_id>&latest=release`
Get latest save: `?app_id=<app_id>&latest=save`

**Pull request merged**